### PR TITLE
Ignore some files in SonarCloud that should not be checked - Closes #4566

### LINKED ISSUE #4566: Ignore some files in SonarCloud that should not be checked
### Description

Some files are being inspected by SonarCloud giving bad scores but they should be ignored by it.

### Motivation

Reduce noise in SonarCloud reports

### Acceptance Criteria

SonarCloud should not check: sql migrations, browser tests templates, reset_mem_tables.sql and protocol-specs folder

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

Some files are being inspected by SonarCloud giving bad scores but they should be ignored by it.

### How did I solve it?

By adding the exclusions to Jenkins Sonar file

### How to manually test it?

- Build should be green
- Sonar should not report issues in the added exclusions

### Review checklist

- [x] The PR resolves #4566
- [x] All new code is covered with unit tests
- [x] Relevant integration / functional tests are added
- [x] Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
- [x] Documentation has been added/updated

### CODE REVIEW

File: Jenkinsfile.sonar [hunk 1]
```diff
@@ -20,7 +20,7 @@ pipeline {
 								sh """#!/bin/bash
 								${scannerHome}/bin/sonar-scanner -Dsonar.projectKey=LiskHQ_lisk-sdk \
 												 -Dsonar.organization=liskhq \
-												 -Dsonar.exclusions=**/test/** \
+												 -Dsonar.exclusions=**/test/**,**/migrations/**,reset_mem_tables.sql,**/protocol-specs/**,**/templates/browsertest.tmpl/** \
```
Comment (fchavant) [inline Jenkinsfile.sonar]: Is this really specific to PRs? If not the same change needs to be made on line 35.

Comment (pablitovicente) [reply Jenkinsfile.sonar]: We can safely ignore them by now in general should I added then on 35?

Comment (fchavant) [reply Jenkinsfile.sonar]: I think it makes sense to have the same exclusion list for PRs (line 23) and branches (line 35)

Comment (pablitovicente) [reply Jenkinsfile.sonar]: Added

**Pull request merged**